enterprise/ha/src/main/java/org/neo4j/kernel/ha/store/StoreUtil.java

```diff
@@ -63,9 +66,30 @@ public static File newBranchedDataDir( File storeDir )
         return result;
     }
 
-    private static boolean isBranchedDataRootDirectory( File file )
+    public static void moveAwayDb( File storeDir, File branchedDataDir, PageCache pageCache ) throws IOException
     {
-        return file.isDirectory() && file.getName().equals( BRANCH_SUBDIRECTORY );
+        for ( File file : relevantDbFiles( storeDir ) )
+        {
+            FileUtils.moveFileToDirectory( file, branchedDataDir );
+        }
+
+        Iterable<FileHandle> handles = pageCache.streamFilesRecursive( storeDir )
+                .filter( fh -> DEEP_STORE_FILE_FILTER.accept( fh.getFile() ) )::iterator;
+        for ( FileHandle handle : handles )
+        {
```

Comment:
here the same thing i would extract for loop content into consumer and pass it into same stream pipeline isntead